backend/runner/schemasync/syncer.go

```diff
@@ -338,3 +340,16 @@ func equalDatabaseMetadata(x, y *storepb.DatabaseMetadata) bool {
 		protocmp.IgnoreFields(&storepb.TableMetadata{}, "row_count", "data_size", "index_size", "data_free"),
 	)
 }
+
+var getCategoryFromCommentReg = regexp.MustCompile("[0-9]+-[0-9]+-[0-9]+")
```

Comment:
Do we need to require the category id should at the beginning of the comment?

Reply:
SGTM